documentation/modules/rql/pages/developer-guide.adoc

```diff
@@ -53,26 +59,27 @@ To create a string representation of the model, we can use the _toString()_ meth
 
 ===== Single argument or body
 
-Do you transport the query via html body or as a single query attribute?
-To parse a query from Rest:
+Do you transport the query via HTML body or as a single query attribute?
```

Comment:
HTML should be http

Reply:
Done -- https://github.com/bci-oss/semanticstack-rql/pull/69/commits/aa243f00cc7d0f034db146dde9b026986b5caa30